sonarts-sq-plugin/its/plugin/src/test/java/org/sonar/typescript/its/ProfileTest.java

```diff
@@ -51,6 +52,9 @@ public void should_raise_issues_for_activated_in_sq_rules() {
     SearchRequest request = new SearchRequest();
     request.setComponentKeys(Collections.singletonList(PROJECT_KEY));
     List<Issue> issuesList = newWsClient().issues().search(request).getIssuesList();
-    assertThat(issuesList).extracting("line").containsExactlyInAnyOrder(1, 2);
+    assertThat(issuesList).extracting(Issue::getLine, Issue::getRule).containsExactlyInAnyOrder(
+      // TODO S3801 ported SonarJS
```

Comment:
This todo will be fixed when you will activate this rule properly in SonarJS, right?

Reply:
no, this rule is not yet implemented for TS in SonarJS https://github.com/SonarSource/SonarJS/issues/1457